app/assets/javascripts/child/docusign.coffee

```diff
@@ -0,0 +1,16 @@
+$ ->
+  return unless $('.show.children').length > 0
+
+  $('#void_button').click ->
+    confirm "
+    Only pending envelopes are voidable via this page.
```

Comment:
The meaning of this first sentence wasn't immediately clear to me, do you think it is necessary? I just wanted to make sure that any user is not confused in any way.